client/index.html

```diff
@@ -218,38 +218,20 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
-									<input type="checkbox" name="join">
-									Show joins
+									<input type="checkbox" name="optional">
+									Show <abbr title="Joins, parts, kicks, nick changes, and mode changes">status messages</abbr>
```

Comment:
I know there were a lot of discussions around options here, and I think the current situation is really nice: show nothing at all, show everything, or show a condensed/human-friendly version. 👏 👏 on getting there!

That being said, there is one state right now that's possible with the checkboxes and has no effect: whenever "Show status messages" is unchecked, "Condense status messages" has no meaning.

- One easy way to fix this would be to make the latter checkbox disabled when the former one is unchecked.
- Another, more meaningful way (but also a tiny bit more involved) would be to make the latter more visually dependent on the former. Something along the lines of:
  <img width="193" alt="screen shot 2017-07-31 at 01 25 59" src="https://user-images.githubusercontent.com/113730/28763874-44167290-758f-11e7-9f98-cbcb48b688c9.png"> <img width="203" alt="screen shot 2017-07-31 at 01 25 38" src="https://user-images.githubusercontent.com/113730/28763876-461a4490-758f-11e7-8123-d3d3517c8dfe.png">
  Of course, with better wording, presentation, etc.
  Any suggestions welcome :)